docs/readmes/lte/dev_notes.md

```diff
@@ -48,6 +48,72 @@ file is generated by mapping the configuration values from `gateway.mconfig` and
 `/etc/systemd/system` directory of the VM at the time of provisioning. You need
 to manually sync changes to these files between guest and host OS.
 
+- Systemd configuration files contain interdependencies of Magma services regarding the start, stop, and restart of services. Therefore, a systemd configuration contains several options to specify these dependencies.
+- The options `Before` and `After` define the order in which services are started and stopped. If a service gets started, the services listed in `Before` and `After` will also be started. If one of the other services gets stopped or its start fails, the service will be stopped. The following dependency graph depicts this order for Magma services.
+
+    ```mermaid
+    graph TD
+        SubscriberDB --> MobilityD
+
+        OVS --> MobilityD
+        MobilityD --> MME
+
+        MobilityD --> PipelineD
+        EnvoyController --> PipelineD
+        EnvoyController --> Envoy
+
+        OVS --> PipelineD
+        PipelineD --> MME
+
+        PipelineD --> ConnectionD
+        PipelineD --> LiAgent
+
+        OVS --> ConnectionD
+        ConnectionD --> MME
+
+        SctpD --> MME
+        SessionD --> MME
+    ```
+
+- The options [Wants](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=) and [Requires](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Requires=) specify a set of services that will be started if the configuring service starts. If the listed services fail to start, this has no impact on the service start in the case of `Wants`. Considering `Requires`, the service will not be started if the option `After` is set on the failing service.
```

Comment:
- This is somewhat hard to understand. How about something similar to the following.
- Would the last part in parenthesis also hold? Or is that non symmetric in this case?

```suggestion
- If a service Alpha [Wants](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=) or [Requires](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Requires=) a service Beta, the service Beta is also started, when service Alpha starts.
  - In case of [Wants](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=) a startup failure of service Beta has no impact on service Alpha.
  - In case of [Requires](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Requires=) a startup failure of service Beta will prevent start of service Alpha, if the option `After` is set on service Beta (or `Before` is set on service `Alpha`).
```

PS.: Applying the suggestion via Github might break the DCO check, though. :shrug:

Reply:
> * Would the last part in parenthesis also hold? Or is that non symmetric in this case?
The behaviour is only given for `After`.